apps/client/src/features/rundown/event-block/EventBlock.module.scss

```diff
@@ -139,7 +143,7 @@ $skip-opacity: 0.2;
 
   .nextTag {
     font-size: 1rem;
-    color: $orange-500;
+    color: $warning-orange;
```

Comment:
I am not sure this is a good change
it is unfortunate that we use the labelled warning colour for this. but that doesnt mean that the next tag is a warning

ie: if we decide to say that all warnings will become purple, should the next tag also become purple?